@payloadcms/plugin-seo 1.1.0 is missing dist/index.js

### Link to reproduction

_No response_

### Describe the Bug

I have updated to the latest version of the plugin (1.10) and the `dist/index.js` file is missing in the package. 

### To Reproduce

Could be checked on NPM

https://www.npmjs.com/package/@payloadcms/plugin-seo?activeTab=code -> dist - index.js is not present

### Payload Version

2.7

### Adapters and Plugins

plugin-seo

Same issue, on a fresh create-payload-app using Postgres.

- npx create-payload-app@latest
- cd payload-blog
- yarn && yarn dev

Error: [nodemon] starting `ts-node src/server.ts -- -I`
unhandledRejection Error: Cannot find module '/Users/xxx/Sites/payload-blog/node_modules/@payloadcms/plugin-seo/dist/index.js'. Please verify that the package.json has a valid "main" entry

--- a/packages/plugin-seo/package.json
+++ b/packages/plugin-seo/package.json
@@ -7,7 +7,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "tsc",
+    "build": "swc ./src -d ./dist --config-file .swcrc",
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "eslint src",
     "lint:fix": "eslint --fix --ext .ts,.tsx src",

--- a/packages/plugin-seo/package.json
+++ b/packages/plugin-seo/package.json
@@ -7,7 +7,10 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "swc ./src -d ./dist --config-file .swcrc",
+    "build": "pnpm copyfiles && pnpm build:swc && pnpm build:types",
+    "build:swc": "swc ./src -d ./dist --config-file .swcrc",
+    "build:types": "tsc --emitDeclarationOnly --outDir dist",
+    "copyfiles": "copyfiles -u 1 \"src/**/*.{html,css,scss,ttf,woff,woff2,eot,svg,jpg,png,json}\" dist/",
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "eslint src",
     "lint:fix": "eslint --fix --ext .ts,.tsx src",

